diesel/src/pg/types/date_and_time/chrono.rs

```diff
@@ -281,7 +281,7 @@ mod tests {
         assert!(query.get_result::<bool>(connection).unwrap());
 
         let max_date = NaiveDate::MAX;
-        let query = select(sql::<Date>("'262143-12-31'::date").eq(max_date));
+        let query = select(sql::<Date>("'262142-12-31'::date").eq(max_date));
```

Comment:
According to the Chrono documentation, NaiveDate::MAX is 262142. I'm guessing this was part of a recent change?